src/index.js

```diff
@@ -94,13 +92,10 @@ export default function rollbarSourcemaps({
           sourcemaps.map((asset) => {
             const form = new FormData()
 
-            form.append('access_token', accessToken)
-            form.append('version', version)
-            form.append('minified_url', `${baseUrl}${asset.original_file}`)
-            form.append('source_map', asset.content, {
-              filename: asset.original_file,
-              contentType: 'application/json'
-            })
+            form.set('access_token', accessToken)
```

Comment:
The main issue I was seeing within our apps using this plugin was that the form data was not being supplied to Rollbar when making the API call from within `uploadSourcemap`, with API responses indicating an access token was missing, and it was definitely defined within our environment so it lead me to find that the FormData was not sending the data correctly via `fetch`. Then I discovered the `node-fetch-native` package [falls back to native fetch](https://github.com/unjs/node-fetch-native#readme) when defined, which doesn't seem compatible with the `form-data` package we are using, which lead me to find https://github.com/form-data/form-data/issues/551 which seems it is not necessary anymore due to being a global in node.